### DISCUSSION TITLE
Chamber Temp/Progress bar

### DISCUSSION THREAD

**Discussion by cnc-lasercraft** [julianschill/klipper-led_effect#191] [Ideas]
Hey all

Would be great to have a something like progress but for a temp sensor. My chamber should be 55 Degrees to start a print. Would be great to have a 'progress'-bar that starts with 0 LED's at a certain temp and all LED's when the target temp is reached.

regards

Urs

**Comment by julianschill** [comment]
Thanks for your suggestion. This is planned as part of #25

**Comment by julianschill** [comment]
This should be possible now in the current release. Use the TemperatureGauge effect
